xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -1500,18 +1609,23 @@ The current depth exceeds the recursive limit set by the maximum depth.</excepti
         <ReturnType>System.Buffers.ReadOnlySequence&lt;System.Byte&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the value of the last processed token as a <see cref="T:System.ReadOnlySpan`1" /> slice of the input payload. If the JSON is provided within a <see cref="T:System.Buffers.ReadOnlySequence`1" /> and the slice that represents the token value fits in a single segment, then <see cref="P:System.Text.Json.Utf8JsonReader.ValueSpan" /> contains the sliced value since it can be represented as a span. Otherwise, <see cref="P:System.Text.Json.Utf8JsonReader.ValueSequence" /> contains the token value.</summary>
+        <summary>Gets the raw value of the last processed token as a ReadOnlySequence&lt;byte&gt; slice of the input payload. If the JSON is provided within a ReadOnlySequence&lt;byte&gt; and the slice that represents the token value fits in a single segment, then <see cref="P:System.Text.Json.Utf8JsonReader.ValueSpan" /> contains the sliced value since it can be represented as a span. Otherwise, <see cref="P:System.Text.Json.Utf8JsonReader.ValueSequence" /> contains the token value.</summary>
```

Comment:
This was incorrect before. ValueSequence returns ReadOnly**Sequence**, not ReadOnly**Span**.

Also, changed it from generic `ROSequence<T>` to the specific `ROSequence<byte>`. However, that loses the link to the ROSequence docs page. Any suggestions?

Reply:
We had a similar problem in Intrinsics, where we needed the documentation to have a link to the API, but the link would not show up if you added a specific type, so what I did was this (look at the summary):

https://docs.microsoft.com/en-us/dotnet/api/system.runtime.intrinsics.vector256.asuint16?view=netcore-3.0#System_Runtime_Intrinsics_Vector256_AsUInt16__1_System_Runtime_Intrinsics_Vector256___0__

Reply:
An alternative is to just leave it as is, since the property value section links to both ROSequence and Byte: 
![image](https://user-images.githubusercontent.com/10886961/64553226-da1e1f00-d2ed-11e9-8f0a-518a6359eaa7.png)

But I'd also move all of the text starting with "If the JSON is provided..." to the Remarks section.

Reply:
> We had a similar problem in Intrinsics, where we needed the documentation to have a link to the API, but the link would not show up if you added a specific type, so what I did was this (look at the summary):

I looked at the summary. I don't understand what you are suggesting here. You are still using the generic type:
![image](https://user-images.githubusercontent.com/6527137/64568588-f59a2180-d30f-11e9-822d-2c58d39b6149.png)

Reply:
@ahsonkhan, there is no link for classes/structs that define the type used in the generic (i.e: there is no link to ReadOnlySequence<byte>). Did you consider using "byte sequence" or "sequence of bytes" as plain text, rather than forcing a link to render? I think that is what @rpetrusha is suggesting.